### PR TITLE
refactor: reorganize TokenRow

### PR DESCRIPTION
**Pull request by cartcrom**
Reduced size of TokenRow.tsx by rewriting parts and breaking out into modules

### CODE REVIEW

File: src/components/Tokens/TokenTable/TokenRow/index.tsx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  percentChange: ReactNode
+  tvl: ReactNode
+  volume: ReactNode
+  sparkline?: ReactNode
+}) {
+  return (
+    <>
+      <ListNumberCell>{index}</ListNumberCell>
+      <NameCell>{name}</NameCell>
+      <PriceCell>
```
Comment (zzmp) [inline src/components/Tokens/TokenTable/TokenRow/index.tsx:144]: Why does `PriceCell` no longer have children?

File: src/components/Tokens/TokenTable/TokenRow/index.tsx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        }
+        price={formatDollar({ num: token.market?.price?.value, isPrice: true, lessPreciseStablecoinValues: true })}
+        percentChange={
+          <>
+            {formattedDelta}
+            {arrow}
+          </>
+        }
+        tvl={formatDollar({ num: token.market?.totalValueLocked?.value })}
+        volume={formatDollar({ num: token.market?.volume?.value })}
```
Comment (zzmp) [inline src/components/Tokens/TokenTable/TokenRow/index.tsx:194]: Is this no longer clickable? Or has that been moved to TokenRowCells? Same with above.

File: src/components/Tokens/TokenTable/TokenTable.tsx [hunk 3]
```diff
@@ -54,14 +60,22 @@ function NoTokensState({ message }: { message: ReactNode }) {
   )
 }
 
+const LoadingRowsWrapper = styled.div`
```
Comment (zzmp) [inline src/components/Tokens/TokenTable/TokenTable.tsx:63]: Is this new?

**Review by zzmp — CHANGES_REQUESTED** [review]

**Comment by tinaszheng** [comment]
@cartcrom are you still working on this? if not can you close 🤔

**Pull request closed without merging**